.github/workflows/rust.yml

```
@@ -2,132 +2,104 @@ name: Rust CI
-          override: true
-      - name: Install cargo-tarpaulin
-        run: cargo install cargo-tarpaulin --version 0.27.3
-      - uses: Swatinem/rust-cache@v2
-      - name: Run tarpaulin
-        run: cargo tarpaulin --all-features --run-types tests --run-types doctests --workspace --out xml -- --test-threads 1
-      - name: Upload to codecov.io
-        uses: codecov/codecov-action@v4
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          fail_ci_if_error: true
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)